pkg/porter/copy.go

```diff
@@ -98,9 +99,27 @@ func (p *Porter) CopyBundle(ctx context.Context, opts *CopyOpts) error {
 
 	bunRef.Reference = destinationRef
 
-	_, err = p.Registry.PushBundle(ctx, bunRef, regOpts)
+	bunRef, err = p.Registry.PushBundle(ctx, bunRef, regOpts)
 	if err != nil {
 		return span.Error(fmt.Errorf("unable to copy bundle to new location: %w", err))
 	}
+
+	if opts.SignBundle {
+		for _, invImage := range bunRef.Definition.InvocationImages {
```

Comment:
The s/InvocationImages/BundleImage is going to be going on for awhile - but I'm not gonna nit over that. Just a comment.

Reply:
Ahh yes, good point. Changed it anyway, to make sure it is not forgotten